.github/workflows/kfp-backend-postgres-tests.yml

```diff
@@ -0,0 +1,245 @@
+name: KFP Backend Postgres Tests
+
+on:
+  workflow_dispatch:
```

Comment:
please see the other CI test triggers (e.g. [here](https://github.com/kubeflow/pipelines/blob/9526abb03d603b0d367fd2f13c8a1949388621a5/.github/workflows/kfp-samples.yml#L3-L7)) and emulate those with the paths set to the files affected, we need them to trigger on pushes to master and PR CI